pytask_io/logger.py

```diff
@@ -1,4 +1,20 @@
 import logging
+import os
 
-logging.basicConfig(level=logging.INFO)
-logger = logging.getLogger(__name__)
+
+def set_log_level():
```

Comment:
I prefer this approach for now, its for low level libraries using pytaskio not really for high level users using this library directly, for that maybe we could add in a kwarg to the main PyTaskIO class..